packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -580,12 +581,15 @@ export default class Calendar extends React.Component {
     if (!this.props.todayButton) {
       return
     }
+
+    const today = getStartOfDate(now(this.props.utcOffset));
```

Comment:
Curious to know why we need `getStartOfDate(now(this.props.utcOffset));` when we can pass `this.state.date` as a parameter to `isDayDisabled` method.

Reply:
@supreethmr `getStartOfDate(now(this.props.utcOffset))` always return today's date, which is what is needed to determine if the Today button should be disabled. `this.state.date` is the selected date which can be any date. Furthermore, `getStartOfDate(now(this.props.utcOffset))` is what is currently being used to set as the selected date when the Today button is clicked so I'm just reusing it here.